app/src/main/kotlin/io/github/feelfreelinux/wykopmobilny/ui/widgets/entry/EntryWidget.kt

```diff
@@ -64,12 +72,12 @@ class EntryWidget(context: Context, attrs: AttributeSet) : CardView(context, att
             text = entry.commentsCount.toString()
 
             setOnClickListener {
-                navigator.openEntryDetailsActivity(getActivityContext()!!, entry.id)
```

Comment:
Use `getActivityContext()` on older apis context from view is not always `Activity`